### PR TITLE
Compatible springboot3

### PR DESCRIPTION
**Pull request by hustchaya**
### Motivation

compatible-springboot3 and add examples

### Modification

Describe the idea and modifications you've done.

### Result

Resolved or fixed #<GitHub issue number>. 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

File: sofa-serverless-runtime/sofa-serverless-adapter-ext/sofa-serverless-adapter-log4j2/src/main/java/com/alipay/sofa/serverless/log4j2/SOFAServerlessLog4j2LoggingSystem.java [hunk 1]
```diff
@@ -312,8 +312,12 @@ private LoggerConfiguration convertLoggerConfig(String name, LoggerConfig logger
 
     @Override
     public void cleanUp() {
-        if (isBridgeHandlerAvailable()) {
-            removeJdkLoggingBridgeHandler();
+        try {
```
Comment (lvjing2) [inline sofa-serverless-runtime/sofa-serverless-adapter-ext/sofa-serverless-adapter-log4j2/src/main/java/com/alipay/sofa/serverless/log4j2/SOFAServerlessLog4j2LoggingSystem.java:315]: 确保 jdk8 单测运行正常的情况下，补充下 jdk 17 的单测。

Comment (hustchaya) [reply sofa-serverless-runtime/sofa-serverless-adapter-ext/sofa-serverless-adapter-log4j2/src/main/java/com/alipay/sofa/serverless/log4j2/SOFAServerlessLog4j2LoggingSystem.java:315]: 直接try catch 不用额外的单侧了?

File: samples/springboot3-samples/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| kafka                  | 模块独立使用 kafka | [samples/springboot3-samples/msg/kafka](https://github.com/sofastack/sofa-serverless/tree/master/samples/springboot3-samples/msg/kafka/)         |
+| mybatis                | 模块使用独立数据源    | [samples/springboot3-samples/db/mybatis](https://github.com/sofastack/sofa-serverless/tree/master/samples/springboot3-samples/db/mybatis)   |
+
+### 注意请在编译器单独导入sofa-serverless/samples/springboot-samples 工程，否则会出现 BizRuntieContext Not found 的报错。
+![biz runtime context not found](bizruntimecontext_not_found.png)
+原因：samples 工程与 sofa-serverless-runtime 在一个工程目录里，会优先使用本地的 sofa-serverless-runtime，而不是 maven 依赖的 sofa-serverless-runtime，导致找不到 BizRuntimeContext 类。
+
+### 支持 Java EE/Jakarta EE
+随着 Oracle 将 Java EE 交给 Eclipse 开始 Java EE 就成为了历史，取而代之的是新的 Jakarta EE 系列。对于开发者而言，最大的改变便是从 jakarta EE 9 开始， EE 相关的类的命名空间从 javax.* 迁移到了 jakarta.*。
+对于这一变更，开源社区的组件也均在不同进度的进行升级适配：例如 Tomcat 从 10.x 版本开始便迁移到了 jakarta 命名空间。而 SpringBoot 3.x 版本也不再支持 Java EE 8，而是基于 Jakarta EE 9 构建。 为了支持 Jakarta EE，sofa-ark 将从 2.2.4 开始发布后缀为 `-jakarta`的包。
```
[the remark this answers is not in the document]
Comment (hustchaya) [reply samples/springboot3-samples/README.md]: 直接cover了 sofa ark就不需要说明了

File: samples/springboot3-samples/db/mybatis/README.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+</dependency>
+<dependency>
+    <groupId>com.alipay.sofa</groupId>
+    <artifactId>sofa-ark-springboot-starter</artifactId>
+    <version>2.2.4-jakarta-SNAPSHOT</version>
+</dependency>
+<dependency>
+    <groupId>com.alipay.sofa</groupId>
+    <artifactId>sofa-ark-support-starter</artifactId>
+    <version>2.2.4-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (hustchaya) [reply samples/springboot3-samples/db/mybatis/README.md:29]: 调整完毕

File: samples/springboot3-samples/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+## SOFAServerless 支持中间件列表与样例
```
[the remark this answers is not in the document]
Comment (hustchaya) [reply samples/springboot3-samples/README.md]: 调整完毕

File: sofa-serverless-runtime/sofa-serverless-adapter-ext/sofa-serverless-adapter-log4j2/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports [hunk 5]
```diff
@@ -0,0 +1 @@
+com.alipay.sofa.serverless.log4j2.SOFAServerlessLog4j2SpringContextListener
```
Comment (yuanyuancin) [inline sofa-serverless-runtime/sofa-serverless-adapter-ext/sofa-serverless-adapter-log4j2/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports:1]: listener也是通过AutoConfiguration.imports文件声明的？

**Review by yuanyuancin — APPROVED** [review]
LGFM

**Review by lvjing2 — APPROVED** [review]
LGTM

**Pull request merged**